core/src/main/scala/com/softwaremill/react/kafka2/DummyConsumer.scala

```diff
@@ -14,14 +15,17 @@ object Streams {
     case Failure(ex) =>
       println("Stream finished with error")
       ex.printStackTrace()
-      as.shutdown()
+      import scala.concurrent.duration._
```

Comment:
there was two different aspects:
- shutdown as on complete
- awaiting for shutdown and reacting to this

Why this is merged together?